service/scaladsl/client/src/main/scala/com/lightbend/lagom/scaladsl/client/ServiceClient.scala

```diff
@@ -119,12 +119,15 @@ trait LagomServiceClientComponents extends TopicFactoryProvider {
   def environment: Environment
   def applicationLifecycle: ApplicationLifecycle
 
+  def configuration: Configuration
```

Comment:
We can also `extends LagomConfigComponent` instead of adding `def configuration: Configuration`. That traitprovides both a play `Configuration` and it's `underlying` typesafe config object (see https://github.com/lagom/lagom/blob/e9f9e82490218756c92cc6e9e9c3d7c116eb81ae/service/scaladsl/api/src/main/scala/com/lightbend/lagom/scaladsl/api/LagomConfigComponent.scala#L9-L12 and https://github.com/lagom/lagom/blob/6cfa96cede049456b10fa76e20a0fb9d1cda2c65/service/scaladsl/client/src/main/scala/com/lightbend/lagom/scaladsl/client/ServiceLocators.scala#L76)

Usage of `Configuration` is a bit inconsistent and I think we should converge on `LagomConfigComponent`.